openedx/features/course_experience/tests/views/test_course_home.py

```diff
@@ -443,6 +444,36 @@ def test_first_purchase_offer_banner_display(self,
         else:
             self.assertNotContains(response, bannerText, html=True)
 
+    @override_waffle_flag(FIRST_PURCHASE_OFFER_BANNER_DISPLAY, active=True)
+    @override_waffle_flag(DISCOUNT_APPLICABILITY_FLAG, active=True)
+    def test_first_purchase_offer_banner_course_requirements(self):
```

Comment:
Please move this to the discounts app. This is a test of the can_receive_discounts function, rather than the display function.